apps/modeler/src/main_window.cpp

```diff
@@ -36,20 +36,18 @@
 
 #include <pcl/apps/modeler/main_window.h>
 
-#include <pcl/apps/modeler/scene_tree.h>
 #include <pcl/apps/modeler/dock_widget.h>
 #include <pcl/apps/modeler/render_window.h>
 #include <pcl/apps/modeler/render_window_item.h>
+#include <pcl/apps/modeler/scene_tree.h>
 
 #include <QFileInfo>
 #include <QSettings>
```

Comment:
Line between qt and vtk headers

Reply:
Applied now automatically by clang-format  ;-)